qa/suites/orch/cephadm/smoke/start.yaml

```diff
@@ -2,6 +2,8 @@ overrides:
   ceph:
     log-ignorelist:
       - MON_DOWN
+      - mons down
+      - mon down
```

Comment:
FYI, there was another variation of this I had saw when testing ignorelist stuff in the cephadm suite https://github.com/ceph/ceph/pull/55765/commits/521931a1ce04641b5d7478530f90331ca2e3df56
It comes out like
```
cluster [WRN] mon.b (rank 2) addr [v2:172.21.15.104:3300/0,v1:172.21.15.104:6789/0] is down (out of quorum)" in cluster log
```

Reply:
Will add this too.